tests/Infra/VersionSyncTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Text.Json;
+using System.Text.RegularExpressions;
+using Xunit;
+
+namespace AzureMcp.Tests.Infra;
+
+public class VersionSyncTests
+{
+    private const string _globalJsonFileName = "global.json";
```

Comment:
That's not the naming convention we use for constants... suggestion: GlobalJsonFileName